src/main/java/com/fasterxml/jackson/databind/deser/BasicDeserializerFactory.java

```diff
@@ -400,7 +390,9 @@ public ValueInstantiator _valueInstantiatorInstance(DeserializationConfig config
      * constructor is to be used: if so, we have implicit names to consider.
      *
      * @since 2.12
+     * @deprecated no longer used, but kept because this protected method might have been overridden(?)
      */
+    @Deprecated
```

Comment:
Actually I'm not sure what to do with this method, since it's no longer used.  Yet I dare not remove since it is `protected`.

Is marking it as deprecated the way to go?

Reply:
Yes, deprecation is the way to go, but as @pjfanning mentioned please always indicate version in which deprecation occurred.